src/scripts/macros/rest-for-the-night.ts

```diff
@@ -113,7 +113,9 @@ export async function restForTheNight(options: ActionDefaultOptions): Promise<Ch
         const withFrequency = actionsAndFeats.filter(
             (a) =>
                 a.frequency &&
-                (a.frequency.per === "day" || Duration.fromISO(a.frequency.per) <= Duration.fromISO("PT8H")) &&
+                (tupleHasValue(["turn", "round", "day"], a.frequency.per) ||
+                    Duration.fromISO(a.frequency.per) <= Duration.fromISO("PT8H")) &&
+                a.frequency.per &&
```

Comment:
if the tupleHasValue and less than check succeeded, why do you have to check for this? Those computations rely on per.